Source/Forms/Controls/Filter.cpp

```diff
@@ -119,6 +119,70 @@ bool Filter::getDisableFilters()
     return ui->checkBoxDisableFilters->isChecked();
 }
 
+void Filter::enableControls(u16 control)
```

Comment:
I added this because IV and PID methods have different/opposite results to filter through. They share no filters so having all the extras visible at a time may not be as straight forward. If they are disabled, have to be able to re-enable them somehow.